rules/libavfilter.json

```diff
@@ -0,0 +1,69 @@
+{
+  "patterns": ["\\blibavfilter\\b"],
+  "dependencies": [
+    {
+      "packages": ["libavfilter-dev"],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "ubuntu"
+        },
+        {
+          "os": "linux",
+          "distribution": "debian"
+        }
+      ]
+    },
+    {
+      "packages": ["libavfilter-free-devel"],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "fedora"
+        }
+      ]
+    },
+    {
+      "packages": ["libavfilter-free-devel"],
+      "pre_install": [
+        {
+          "command": "yum install -y epel-release"
+        }
+      ],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "rockylinux",
+          "versions": ["9"]
+        }
+      ]
+    },
+    {
+      "packages": ["ffmpeg-devel"],
+      "pre_install": [
+        { "command": "yum install -y epel-release" },
+        { "command": "yum install -y --nogpgcheck https://mirrors.rpmfusion.org/free/el/rpmfusion-free-release-$(rpm -E %rhel).noarch.rpm https://mirrors.rpmfusion.org/nonfree/el/rpmfusion-nonfree-release-$(rpm -E %rhel).noarch.rpm" }
+      ],
```

Comment:
Similar note from opencv here - some of the EPEL packages on Rocky Linux 8 seem to require dependencies from PowerTools:
```sh
$ yum install -y ffmpeg-devel
Last metadata expiration check: 0:00:06 ago on Mon Mar 13 18:43:46 2023.
Error: 
 Problem: package ffmpeg-devel-4.2.8-1.el8.x86_64 requires libavdevice(x86-64) = 4.2.8-1.el8, but none of the providers can be installed
  - package ffmpeg-devel-4.2.8-1.el8.x86_64 requires libavdevice.so.58()(64bit), but none of the providers can be installed
  - conflicting requests
  - nothing provides libSDL2-2.0.so.0()(64bit) needed by libavdevice-4.2.8-1.el8.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```
Splitting centos8 out into a separate entry with PowerTools should work, e.g. https://github.com/rstudio/r-system-requirements/blob/4921b8c2d1d5c5a2e6b4e4415c11bda21b7a9bcc/rules/gdal.json#L95-L113